include/xeus-python/xinterpreter.hpp

```diff
@@ -66,7 +69,7 @@ namespace xpyt
 
         nl::json internal_request_impl(const nl::json& content) override;
 
-        void redirect_output();
+        void configure_redirect_output();
```

Comment:
I am +1 with the feature but I would keep the current method name for backward compatibility - even though it is probably ok given the adoption.